src/transformers/onnx/config.py

```diff
@@ -325,7 +348,8 @@ def generate_dummy_inputs(
                 seq_length, fixed_dimension=OnnxConfig.default_fixed_sequence, num_token_to_add=token_to_add
             )
             # Generate dummy inputs according to compute batch and sequence
-            dummy_input = [" ".join([preprocessor.unk_token]) * seq_length] * batch_size
+            input_token = preprocessor.unk_token if preprocessor.unk_token else "0"
```

Comment:
```suggestion
            input_token = preprocessor.unk_token if preprocessor.unk_token is not None else "0"
```
We don't rely on Python bool magic conversion in the library, so let's test explicitly.

Reply:
Done, Added explicit tests for None and empty string